pulp_service/pulp_service/app/authorization.py

```diff
@@ -23,33 +25,50 @@ def has_permission(self, request, view):
         # Admins have all permissions
         if request.user.is_superuser:
             return True
+
+        domain_name = request.META['PATH_INFO'].removeprefix(settings.API_ROOT).split("/")[0]
```

Comment:
isn't there a get_domain already? I'm totally unsure

Reply:
yes there is ... i will try using that here.